### PR TITLE
Animate match

### PR DESCRIPTION
**Pull request by HelenHMaher**
created a TopNav that should be reusable in all the places that it occurs if you pass the appropriate props! :)

### CODE REVIEW

File: src/components/BottomNav/BottomNav.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { Thunder, Heart, Star } from '../styleElements/icons';
```
Comment (HelenHMaher) [inline src/components/BottomNav/BottomNav.js:2]: I keep noticing little things that need to be updated based on the Figma changes

File: src/components/Dashboard/Dashboard.js [hunk 2]
```diff
@@ -5,7 +5,8 @@ import Friends from '../Friends/Friends';
 import { MatchPage } from '../MatchPage';
 import { Profile } from '../Profile/Profile';
 //import { ProfileEdit } from '../Profile/ProfileEdit';
-import { Start } from '../Start/Start';
+import { Start } from '../Start';
+import { FilterPage } from '../FilterPage';
```
Comment (HelenHMaher) [inline src/components/Dashboard/Dashboard.js:9]: added a link for the FilterPage (which I still need to code)

File: src/components/MatchPage/MatchCard/MatchCard.js [hunk 3]
```diff
@@ -69,7 +69,7 @@ export const MatchCard = ({ decision, reset }) => {
 
   //when a decision is made it triggers an axios call
 
-  const serverURL = 'http://localhost:5000';
+  const serverURL = process.env.REACT_APP_SERVER;
```
Comment (HelenHMaher) [inline src/components/MatchPage/MatchCard/MatchCard.js:72]: this is me importing the .env variable

File: src/components/MatchPage/MatchPage.css [hunk 4]
```diff
@@ -12,6 +11,7 @@
   flex-direction: column;
   align-items: center;
   justify-content: center;
+  height: 800px;
```
Comment (HelenHMaher) [inline src/components/MatchPage/MatchPage.css:14]: just fixing some minor behaviors

File: src/components/MatchPage/MatchPage.js [hunk 5]
```diff
@@ -44,6 +46,8 @@ export function MatchPage() {
 
   return (
     <div className="matchPage">
+      <CirclesBackground />
```
Comment (HelenHMaher) [inline src/components/MatchPage/MatchPage.js:49]: adding the TopNav and the new Figma designs have the circlesBackground in the MatchPage now

File: src/components/TopNav/index.js [hunk 6]
```diff
@@ -0,0 +1,7 @@
+//will display the ChevronLeft icon on the left if "backIcon" is true
```
Comment (HelenHMaher) [inline src/components/TopNav/index.js:1]: this page should explain how to use the props to make the TopNav be reusable but let me know if something doesn't make sense or it doesn't work for you

File: src/components/styleElements/buttons/ShotsButton/ShotsButton.css [hunk 7]
```diff
@@ -1,6 +1,7 @@
 .shotsButton {
   width: 92px;
   height: 92px;
+  cursor: pointer;
```
Comment (HelenHMaher) [inline src/components/styleElements/buttons/ShotsButton/ShotsButton.css:4]: ugh, more annoying CSS details I keep missing and fix as I notice them

File: src/components/styleElements/icons/ChevronLeft.js [hunk 8]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import './Icon.css';
 
-export function ChevronLeft({ size }) {
+export function ChevronLeft({ size, className }) {
```
Comment (HelenHMaher) [inline src/components/styleElements/icons/ChevronLeft.js:4]: I have noticed that some of the icons require very specific colors that are not either of the default text colors so I had to adjust this one to take a prop that will let me change it's color to be more specific

File: src/components/styleElements/icons/index.js [hunk 9]
```diff
@@ -9,6 +9,8 @@ export { Star } from './Star';
 //square icons so size is used as both width and height
 //accepts string "className" to select a className that will determine stroke color
 export { Flip } from './Flip';
+export { ChevronLeft } from './ChevronLeft';
```
Comment (HelenHMaher) [inline src/components/styleElements/icons/index.js:12]: just updating the icon/index.js to reflect the changes to the ChevronLeft props and the new Filter icon

**Review by HelenHMaher — COMMENTED** [review]
everything seems to be working and shouldn't confict with anyone else's code

**Review by FilipNew1234 — APPROVED** [review]
Approved like a mf! 

But I did check out all the changes, nothing seems dangerous

**Pull request merged**